src/index.css

```diff
@@ -15,6 +11,13 @@ body, #root, html {
   overflow: visible;
 }
 
+html {
+  font-size: 62.5%; /* Now 10px = 1rem! */
+  scroll-snap-type: y proximity;
+  scroll-padding-top: 120px;
+  overflow-y: scroll;
```

Comment:
as we talked about in slack, we should try to move the `120px` (size of the nav) to a place that logically should have knowledge of that, and use the variables to get it.

also `overflow-y: scroll;` would ideally be `overflow: auto;`, hopefully that doesn't break everything 😬

Reply:
I think `overflow: auto` should work fine... I'll give it a try. And move the offset to the main content component as discussed.

Reply:
Seems I can put `overflow: auto` in the `MainContentWrapper` and it works fine. This is also right below the one element that really needs `overflow: visible` (the `div#root`)

Reply:
But... hopefully this doesn't break mobile *looks at the monster layout in Content.tsx*

Reply:
Ignore that last one, `overflow: auto` needs to be in the html tag and that seems to just work.